### PR TITLE
578-bad-words-script-for-g11

### PR DESCRIPTION
**Pull request by benvand**
* Refactor bad words script to be a pipeline
  * check out relevant directories
  * run script
  * publish report
  * notify slack

### CODE REVIEW

File: job_definitions/scan_g_cloud_services_for_bad_words.yml [hunk 1]
```diff
@@ -4,42 +4,63 @@
...
+            sh("cp -r digitalmarketplace-frameworks ./data/digitalmarketplace-frameworks")
+          }
+          stage('Clone and copy bad words') {
+            dir('digitalmarketplace-bad-words') {
+              git url: 'git@github.com:alphagov/digitalmarketplace-bad-words.git', branch: 'master', credentialsId: 'github_com_and_enterprise'
+            }
+            sh("cp -r digitalmarketplace-bad-words ./data/digitalmarketplace-bad-words")
+          }
+          stage('Run bad words script') {
+            sh('docker run -e DM_DATA_API_TOKEN_{{ environment|upper }} --volume \$(pwd)/data:/app/data digitalmarketplace/scripts ./scripts/framework-applications/scan-g-cloud-services-for-bad-words.py "{{ environment }}" /app/data/digitalmarketplace-frameworks /app/data/digitalmarketplace-bad-words/blacklist.txt $FRAMEWORK /app/data')
```
Comment (lfdebrux) [inline job_definitions/scan_g_cloud_services_for_bad_words.yml]: Can we split this command over multiple lines to make it easier to read?

File: job_definitions/scan_g_cloud_services_for_bad_words.yml [hunk 2]
```diff
@@ -4,42 +4,63 @@
...
+            dir('digitalmarketplace-frameworks') {
+              git url: 'git@github.com:alphagov/digitalmarketplace-frameworks.git', branch: 'master'
+            }
+            sh("cp -r digitalmarketplace-frameworks ./data/digitalmarketplace-frameworks")
+          }
+          stage('Clone and copy bad words') {
+            dir('digitalmarketplace-bad-words') {
+              git url: 'git@github.com:alphagov/digitalmarketplace-bad-words.git', branch: 'master', credentialsId: 'github_com_and_enterprise'
+            }
+            sh("cp -r digitalmarketplace-bad-words ./data/digitalmarketplace-bad-words")
```
Comment (lfdebrux) [inline job_definitions/scan_g_cloud_services_for_bad_words.yml]: Is there any reason not to clone into data/digitalmarketplace-bad-words?

**Comment by lfdebrux** [comment]
Looks good to me, some questions but nothing majorly important

File: job_definitions/scan_g_cloud_services_for_bad_words.yml (hunk 1 above)
Comment (benvand) [reply job_definitions/scan_g_cloud_services_for_bad_words.yml]: Done 👍

File: job_definitions/scan_g_cloud_services_for_bad_words.yml (hunk 2 above)
Comment (benvand) [reply job_definitions/scan_g_cloud_services_for_bad_words.yml]: Done 👍

**Pull request merged**